### PR TITLE
Adds Further Reading Links to certain Codespaces docs

### LINKED ISSUE #9906: Add "Further Reading links" between end-user and org guides for Codespaces
We have two sections on managing your Codespaces: [one for developers managing their own codespaces](https://docs.github.com/en/codespaces/managing-your-codespaces/) and [one for Org maintainers managing policies for their org](https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/managing-encrypted-secrets-for-your-repository-and-organization-for-codespaces). 

We should add some additional cross-linking between the following docs under a "Further Reading" section:

Org guides:

- [Managing encrypted secrets for your repository and organization for Codespaces](https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/managing-encrypted-secrets-for-your-repository-and-organization-for-codespaces) -> Managing encrypted secrets for your codespaces
- [Managing repository access for your organization's codespaces](https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/managing-repository-access-for-your-organizations-codespaces) -> Managing repository access for your codespaces
- [Reviewing your organization's audit logs for Codespaces](https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/reviewing-your-organizations-audit-logs-for-codespaces) - >Reviewing your security logs for Codespaces

User guides:
- [Managing encrypted secrets for your codespaces](https://docs.github.com/en/codespaces/managing-your-codespaces/managing-encrypted-secrets-for-your-codespaces) - > Managing encrypted secrets for your repository and organization for Codespaces
- [Managing repository access for your codespaces](https://docs.github.com/en/codespaces/managing-your-codespaces/managing-repository-access-for-your-codespaces) -> Managing repository access for your organization's codespaces
- [Reviewing your security logs for Codespaces](https://docs.github.com/en/codespaces/managing-your-codespaces/reviewing-your-security-logs-for-codespaces) -> Reviewing your organization's audit logs for Codespaces

### PR DESCRIPTION
**Pull request by thispsj**
### Why:

Closes #9906 

### What's being changed:

Done the cross linking as mentioned in the issue linked above.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [X] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@thispsj 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/codespaces/managing-codespaces-for-your-organization/managing-encrypted-secrets-for-your-repository-and-organization-for-codespaces.md [hunk 1]
```diff
@@ -76,3 +76,7 @@ You can check which access policies are applied to a secret in your organization
 1. The list of secrets includes any configured permissions and policies. For example:
 ![Secrets list](/assets/images/help/settings/actions-org-secrets-list.png)
 1. For more details on the configured permissions for each secret, click **Update**.
+
+## Further Reading
+
+  * [Managing encrypted secrets for your codespaces.](/codespaces/managing-your-codespaces/managing-encrypted-secrets-for-your-codespaces)
```
Comment (ethanpalm) [inline content/codespaces/managing-codespaces-for-your-organization/managing-encrypted-secrets-for-your-repository-and-organization-for-codespaces.md:82]:
```suggestion
- "[Managing encrypted secrets for your codespaces](/codespaces/managing-your-codespaces/managing-encrypted-secrets-for-your-codespaces)"
```

File: content/codespaces/managing-codespaces-for-your-organization/managing-repository-access-for-your-organizations-codespaces.md [hunk 2]
```diff
@@ -28,3 +28,7 @@ To manage which users in your organization can use {% data variables.product.pro
   ![Radio buttons to manage trusted repositories](/assets/images/help/settings/codespaces-org-access-and-security-radio-buttons.png)
 1. If you chose "Selected repositories", select the drop-down menu, then click a repository to allow the repository's codespaces to access other repositories owned by your organization. Repeat for all repositories whose codespaces you want to access other repositories.
     !["Selected repositories" drop-down menu](/assets/images/help/settings/codespaces-access-and-security-repository-drop-down.png)
+
+## Further Reading
+
+  * [Managing repository access for your codespaces.](/codespaces/managing-your-codespaces/managing-repository-access-for-your-codespaces)
```
Comment (ethanpalm) [inline content/codespaces/managing-codespaces-for-your-organization/managing-repository-access-for-your-organizations-codespaces.md:34]:
```suggestion
- "[Managing repository access for your codespaces](/codespaces/managing-your-codespaces/managing-repository-access-for-your-codespaces)"
```

File: content/codespaces/managing-codespaces-for-your-organization/reviewing-your-organizations-audit-logs-for-codespaces.md [hunk 3]
```diff
@@ -19,3 +19,7 @@ When any member of your organization performs an action related to {% data varia
 ![Audit log with Codespaces information](/assets/images/help/settings/codespaces-audit-log-org.png)
 
 The audit log includes details such as who performed the action, what the action was, and when the action was performed. For information on {% data variables.product.prodname_codespaces %} actions, see "[{% data variables.product.prodname_codespaces %} category actions](/organizations/keeping-your-organization-secure/reviewing-the-audit-log-for-your-organization#codespaces-category-actions)."
+
+## Further Reading
+
+  * [Reviewing your organization's audit logs for Codespaces.](/codespaces/managing-your-codespaces/reviewing-your-security-logs-for-codespaces)
```
Comment (ethanpalm) [inline content/codespaces/managing-codespaces-for-your-organization/reviewing-your-organizations-audit-logs-for-codespaces.md:25]:
```suggestion
- "[Reviewing your security logs for {% data variables.product.prodname_codespaces %}](/codespaces/managing-your-codespaces/reviewing-your-security-logs-for-codespaces)"
```

File: content/codespaces/managing-your-codespaces/managing-encrypted-secrets-for-your-codespaces.md [hunk 4]
```diff
@@ -80,3 +80,7 @@ You can update the value of an existing secret, and you can change which reposit
   !["Delete" button](/assets/images/help/settings/codespaces-secret-delete-button.png)
 1. Read the warning, then click **OK**.
   ![Confirmation to delete secret](/assets/images/help/settings/codespaces-secret-delete-warning.png)
+
+## Further Reading
+
+  * [Managing encrypted secrets for your repository and organization for Codespaces.](/codespaces/managing-codespaces-for-your-organization/managing-encrypted-secrets-for-your-repository-and-organization-for-codespaces)
```
Comment (ethanpalm) [inline content/codespaces/managing-your-codespaces/managing-encrypted-secrets-for-your-codespaces.md:86]:
```suggestion
- "[Managing encrypted secrets for your repository and organization for {% data variables.product.prodname_codespaces %}](/codespaces/managing-codespaces-for-your-organization/managing-encrypted-secrets-for-your-repository-and-organization-for-codespaces)"
```

File: content/codespaces/managing-your-codespaces/managing-repository-access-for-your-codespaces.md [hunk 5]
```diff
@@ -22,3 +22,7 @@ When you enable access and security for a repository owned by your user account,
   ![Radio buttons to manage trusted repositories](/assets/images/help/settings/codespaces-access-and-security-radio-buttons.png)
 1. If you chose "Selected repositories", select the drop-down menu, then click a repository to allow the repository's codespaces to access other repositories you own. Repeat for all repositories whose codespaces you want to access other repositories you own.
   !["Selected repositories" drop-down menu](/assets/images/help/settings/codespaces-access-and-security-repository-drop-down.png)
+
+## Further Reading
+
+  * [Managing repository access for your organization's codespaces.](/codespaces/managing-codespaces-for-your-organization/managing-repository-access-for-your-organizations-codespaces)
```
Comment (ethanpalm) [inline content/codespaces/managing-your-codespaces/managing-repository-access-for-your-codespaces.md:28]:
```suggestion
- "[Managing repository access for your organization's codespaces](/codespaces/managing-codespaces-for-your-organization/managing-repository-access-for-your-organizations-codespaces)"
```

File: content/codespaces/managing-your-codespaces/reviewing-your-security-logs-for-codespaces.md [hunk 6]
```diff
@@ -20,3 +20,7 @@ When you perform an action related to {% data variables.product.prodname_codespa
 ![security log with Codespaces information](/assets/images/help/settings/codespaces-audit-log.png)
 
 The security log includes details on what action occurred and when you performed it. For information about {% data variables.product.prodname_codespaces %} actions, see "[{% data variables.product.prodname_codespaces %} category actions](/github/authenticating-to-github/reviewing-your-security-log#codespaces-category-actions)".
+
+## Further Reading
+
+  * [Reviewing your organization's audit logs for Codespaces.](/codespaces/managing-codespaces-for-your-organization/reviewing-your-organizations-audit-logs-for-codespaces)
```
Comment (ethanpalm) [inline content/codespaces/managing-your-codespaces/reviewing-your-security-logs-for-codespaces.md:26]:
```suggestion
- "[Reviewing your organization's audit logs for {% data variables.product.prodname_codespaces %}](/codespaces/managing-codespaces-for-your-organization/reviewing-your-organizations-audit-logs-for-codespaces)"
```

**Review by ethanpalm — DISMISSED** [review]
👋🏼  Thanks for creating this PR, @thispsj! I've just made a few small suggestions your changes to bring them in line with our style guide. The main change is that for further reading links, we put them in quotation marks if it is a link to another article in the GitHub docs.

I'll add these changes and this merged once all the checks are passing 😄 

Thanks for taking the time to make these changes and help improve the GitHub docs! 🎉

**Review by ethanpalm — APPROVED** [review]

**Comment by Dstiers1191** [comment]
d

Derek stiers

**Pull request merged**